src/Pages/Home/Features/Features.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styles from '../home.module.css';
+
+function Features() {
+  return (
+    <>
+
+      <div className={`row ${styles.featurette}`}>
+        <div className="col-md-8 order-md-2">
+          <h2 className={`${styles.featurette_heading} fw-normal lh-1`}>
+
+            We offers a large hall, studio and a small community meeting room
+            {' '}
+            <span className={styles.text_main}>Available For Rent.</span>
```

Comment:
[Optional] You can put "Available for rent" in a link that takes the user to the halls page, in my opinion, this could provide a good user experience, especially for mobile users.